app/app.go

```diff
@@ -454,10 +452,46 @@ func NewApp(
 	return app
 }
 
+func (app *AkashApp) registerUpgradeHandlers() {
+	app.keeper.upgrade.SetUpgradeHandler("v0.43", func(ctx sdk.Context, plan upgradetypes.Plan,
```

Comment:
Please suggest a suitable name for the upgrade handler. Currently, it is named `v0.43` which is based on the cosmos version we are upgrading to. Should it be something like: `v0.13` as that will be the version of the next akash  release?
Or should it be something like: `akashnet-2-upgrade-1` as it was previously?

Reply:
can we do it using this pattern `akash_v0.13.0_cosmos_v0.43.0` as `-` is valid symbol in semantic versioning indicating it is prerelease

Reply:
Done